go/src/koding/api/apitest/apitest.go

```diff
@@ -31,20 +36,33 @@ type HTTPIdleConnectionsCloser interface {
 	CloseIdleConnections()
 }
 
+// FakeAuth is a mock which provides fake authentication server.
+//
+// It implements both client-facing interface  the Auth method -
+// than can be used with clients' transports, and also a server-facing
+// one - the GetSession method - which is a handler that can be
+// used with http server.
 type FakeAuth struct {
 	Sessions map[string]*api.Session
 
 	mu sync.RWMutex
 }
 
-var _ http.Handler = http.HandlerFunc((&FakeAuth{}).GetSession)
+var (
+	_ http.Handler = http.HandlerFunc((&FakeAuth{}).GetSession)
```

Comment:
This one is probably not needed. If `GetSession` is not of `http.HandlerFunc` signature, transport tests won't compile.